### PR TITLE
Add a new class

### PR DESCRIPTION
**Pull request by IsmailEmira**

### CODE REVIEW

**Comment by MartaTaran01** [comment]
looks good

**Comment by DianaSvitukha** [comment]
looks good

**Comment by sdfiq** [comment]
looks good

File: src/week1/Ismail.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package week1;
+
+public class Ismail {
+
+
+}
```
Comment (sdfiq) [inline src/week1/Ismail.java:6]: its fine

**Review by sdfiq — APPROVED** [review]

**Review by zufarovs — APPROVED** [review]

**Pull request merged**